portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -76,4 +83,16 @@ private String convertToJsonUsingGson(List<String> messages) {
     String json = gson.toJson(messages);
     return json;
   }
+
+  private int getNumComments(HttpServletRequest request) {
+      String userInput = request.getParameter("max");
+      int numComments;
+      try {
+        numComments = Integer.parseInt(userInput);
+      } catch (NumberFormatException e) {
+        System.err.println("Could not convert to int: " + userInput);
+        return 5;
```

Comment:
Why do we return 5 here?

Reply:
I just used 5 as a default value so it sets to if the user doesn't select anything